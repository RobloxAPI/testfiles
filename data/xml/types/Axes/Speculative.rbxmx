<roblox version="4">
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ComponentsUpperBool</string>
			<Axes name="Axes">
				<X>true</X>
				<Y>false</Y>
				<Z>true</Z>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ComponentsLowerBool</string>
			<Axes name="Axes">
				<x>true</x>
				<y>false</y>
				<z>true</z>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ComponentsUpperNumber</string>
			<Axes name="Axes">
				<X>1</X>
				<Y>0</Y>
				<Z>1</Z>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ComponentsLowerNumber</string>
			<Axes name="Axes">
				<x>1</x>
				<y>0</y>
				<z>1</z>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SubcomponentsUpperBool</string>
			<Axes name="Axes">
				<axes>
					<X>true</X>
					<Y>false</Y>
					<Z>true</Z>
				</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SubcomponentsLowerBool</string>
			<Axes name="Axes">
				<axes>
					<x>true</x>
					<y>false</y>
					<z>true</z>
				</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SubcomponentsUpperNumber</string>
			<Axes name="Axes">
				<axes>
					<X>1</X>
					<Y>0</Y>
					<Z>1</Z>
				</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SubcomponentsLowerNumber</string>
			<Axes name="Axes">
				<axes>
					<x>1</x>
					<y>0</y>
					<z>1</z>
				</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ListSpaceBool</string>
			<Axes name="Axes">true false true</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ListSpaceNumber</string>
			<Axes name="Axes">1 0 1</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ListSpaceComponent</string>
			<Axes name="Axes">X Z</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ListCommaBool</string>
			<Axes name="Axes">true, false, true</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ListCommaNumber</string>
			<Axes name="Axes">1, 0, 1</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">ListCommaComponent</string>
			<Axes name="Axes">X, Z</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SublistSpaceBool</string>
			<Axes name="Axes">
				<axes>true false true</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SublistSpaceNumber</string>
			<Axes name="Axes">
				<axes>1 0 1</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SublistSpaceComponent</string>
			<Axes name="Axes">
				<axes>X Z</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SublistCommaBool</string>
			<Axes name="Axes">
				<axes>true, false, true</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SublistCommaNumber</string>
			<Axes name="Axes">
				<axes>1, 0, 1</axes>
			</Axes>
		</Properties>
	</Item>
	<Item class="ArcHandles">
		<Properties>
			<string name="Name">SublistCommaComponent</string>
			<Axes name="Axes">
				<axes>X, Z</axes>
			</Axes>
		</Properties>
	</Item>
</roblox>
