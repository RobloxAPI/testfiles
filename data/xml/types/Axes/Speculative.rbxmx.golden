{
	"Format": "rbxmx",
	"Output": "model",
	"Data": {
		"Metadata": [],
		"Instances": [
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 0,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ComponentsUpperBool"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 1,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ComponentsLowerBool"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 2,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ComponentsUpperNumber"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 3,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ComponentsLowerNumber"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 4,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SubcomponentsUpperBool"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 5,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SubcomponentsLowerBool"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 6,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SubcomponentsUpperNumber"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 7,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SubcomponentsLowerNumber"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 8,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ListSpaceBool"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 9,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ListSpaceNumber"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 10,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ListSpaceComponent"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 11,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ListCommaBool"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 12,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ListCommaNumber"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 13,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "ListCommaComponent"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 14,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SublistSpaceBool"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 15,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SublistSpaceNumber"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 16,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SublistSpaceComponent"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 17,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SublistCommaBool"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 18,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SublistCommaNumber"
					}
				],
				"Children": []
			},
			{
				"ClassName": "ArcHandles",
				"IsService": false,
				"Reference": 19,
				"Properties": [
					{
						"Name": "Axes",
						"Type": "Axes",
						"Value": {
							"X": false,
							"Y": false,
							"Z": false
						}
					},
					{
						"Name": "Name",
						"Type": "String",
						"Value": "SublistCommaComponent"
					}
				],
				"Children": []
			}
		]
	}
}